src/actions/index.ts

```diff
@@ -0,0 +1,10 @@
+export enum ActionTypeEnum {
```

Comment:
Maybe we're mixed Typescript & javascript type here. `enum` is not a type in Javascript, isn't it?
We can use `const` instead.
So this one can update to
```
export const ActionTypes
```